application/container-builder/internal/containers/buildpack/builder.go

```diff
@@ -0,0 +1,104 @@
+package image
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	pack "github.com/buildpacks/pack/pkg/client"
+	"github.com/hashicorp/go-hclog"
```

Comment:
you would not have known but I was thinking maybe all go serices could use https://github.com/rs/zerolog ? what do you think? it was this or apex in my mind.